content/build-tool.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Build system
+title: Build System
```

Comment:
Can we get agreement on style here. We have been making all titles only first word capitalized in <h> tag. Now folks are wanting to change it back. I personally don't care, but it would be best to stay with one approach and everything for the most part is only capitalizing the first word.